doc/components_1.md

```diff
@@ -27,15 +28,97 @@ For instance, the `mj-button` component is, on the inside, a complex HTML layout
+  HTML also supports "hsl()" notation and others. Still, they operate similarly. -->
+<!--- Example 2:
+  MJML's "mj-text" > "padding" limits the attribute values to "unit(px,%,)".
+  HTML's "padding" also supports "em" units and
+  three <em>global value</em> strings. Still similar. -->
+  
+The _unit_ column specifies whether the attribute must be, for instance,
+  a string, a number, or other unit.
+There's a little more on this later in this section.
+
+The _description_ column explains the purpose of the attribute
+  and what the developer can achieve with it.
+Sometimes, it indicates equivalence with the same-name HTML attribute. 
+
+The _default value_ column specifies the default value for the attribute.
+Attributes without a default value have _n/a_ here.
+
+<!--- Not ambiguous because MJML does not support "n/a" as any attribute value. --> 
+
+<aside class="notice">
+MJML syntax for <em>HTML</em> attributes differs from syntax for <em>MJML</em> attributes.
+MJML "css-class" attributes introduce HTML attributes in MJML components;
+  they refer to [mj-style](#mj-style) definitions.
+MJML attributes use the same syntax in MJML tags as HTML attributes in HTML tags.
+</aside>
+
+<aside class="notice">
+MJML can support both an HTML attribute and an MJML attribute
+  with the same spelling and different values.
+In such code, MJML uses the HTML attribute
+  inside an MJML "css-class" attribute and
+  in raw HTML inside a "class" attribute.
+It uses the MJML attribute otherwise.
+</aside>
+
+<!--- https://mjml.io/try-it-live/H1LbbsI1I -->
+
+<aside class="notice">
+Use caution in choosing attributes and attribute values for email.
+Some widely used email clients don't support
+  some popular attributes or some attribute values.
+Some email clients render a given attribute or attribute value
+  differently than other clients. 
+As [one Email Design Reference](https://templates.mailchimp.com/)
+  (accessed 2019-12-23) said, "Coding an HTML email isn't much different than
+  coding a website was back in the late '90s."
+Ouch!
+</aside>
+
+
 ## mjml
 
-A MJML document starts with a `<mjml>` tag, it can contain only `mj-head` and `mj-body` tags. Both have the same purpose of `head` and `body` in a HTML document.
+An MJML document starts with an `<mjml>` tag.
+It can contain only `<mj-head>` and `<mj-body>` tags.
+Both have the same purpose of `<head>` and `<body>` in an HTML document.
 
 attribute | unit | description | default value
 ----------|------|-------------|---------------
-owa | string | if set to "desktop", will force desktop display on outlook. Otherwise outlook will display mobile version since it ignores media queries | none
-lang | string | used as `<html lang="">` attribute | none
+owa | string | If set to "desktop", will force desktop display on outlook. Otherwise outlook will display mobile version since it ignores media queries | n/a
+lang | string | same as HTML's `<html lang="">` | n/a
 
+<aside class="notice">
```

Comment:
Here I don't think this is necessary the description part in the md table is enough

Reply:
I'll change as you like. Maybe I only made minimal changes here to the existing? Dunno what to change.

Reply:
The whole aside block imo isn't really relevant as you have multiple example of MJML in the doc

Reply:
Well, I thought at one point that this web page was updating line numbers for new commits. Maybe not.

Maybe this web page is now telling me the offending `aside` is at line 38. I'm unable to find any commit, let alone in this time frame with an `aside` there.

The nearest `aside` I find is the one with opening text 
>MJML syntax for <em>HTML</em> attributes differs from syntax for ...

We've deleted that one.